var/spack/repos/builtin/packages/py-pytest-allclose/package.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2013-2024 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyPytestAllclose(PythonPackage):
+    """Pytest fixture extending Numpy's allclose function."""
+
+    pypi = "pytest-allclose/pytest-allclose-1.0.0.tar.gz"
+
+    maintainers("paugier")
+
+    license("MIT", checked_by="paugier")
+
+    # fmt: off
+    version("1.0.0", sha256="b2f0c521fa652281400d4a105c84454db3c50b993bcfee9861380be69cc6b041")
+    # fmt: on
+
+    extends("python@3.9:", type=("build", "run"))
+    depends_on("py-setuptools", type="build")
```

Comment:
```suggestion
    depends_on("py-setuptools@:63", type="build")
```

Reply:
Also requires `py-numpy>=1.11` and `py-pytest`.

Reply:
Is it fine to write `depends_on("py-numpy>=1.11", type="run")` or should I use a notation with `@`?

Reply:
It should be fine now.